core/templates/pages/exploration-editor-page/services/exploration-rights.service.ts

```diff
@@ -230,6 +230,17 @@ export class ExplorationRightsService {
   }
 
   removeRoleAsync(memberUsername: string): Promise<void> {
+    const categories = ['ownerNames', 'editorNames', 'viewerNames'] as const;
+    let initialUsernamesByCategory: string[];
+    type Category = (typeof categories)[number];
+    let userRole: Category;
```

Comment:
Suggest `roleOfRemovedUser` (for clarity)

Reply:
Done